src/main/java/edu/greenblitz/robotName/commands/arm/roller/RunRoller.java

```diff
@@ -0,0 +1,21 @@
+package edu.greenblitz.robotName.commands.arm.roller;
+
+import edu.greenblitz.robotName.subsystems.arm.EndEffector.RollerUtils.RollerConstants;
+import edu.greenblitz.robotName.subsystems.arm.Wrist;
+
+public class RunRoller extends RollerCommand{
```

Comment:
i would add to both classes a time constant for safty. add to the is finiehd once the object is out, wait half a sec or smth to make sure its actually out